test/EFCore.PG.FunctionalTests/Query/NullSemanticsQueryNpgsqlTest.cs

```diff
@@ -6,9 +6,143 @@ namespace Npgsql.EntityFrameworkCore.PostgreSQL.Query;
 // ReSharper disable once UnusedMember.Global
 public class NullSemanticsQueryNpgsqlTest : NullSemanticsQueryTestBase<NullSemanticsQueryNpgsqlFixture>
 {
-    public NullSemanticsQueryNpgsqlTest(NullSemanticsQueryNpgsqlFixture fixture)
+    public NullSemanticsQueryNpgsqlTest(NullSemanticsQueryNpgsqlFixture fixture, ITestOutputHelper testOutputHelper)
         : base(fixture)
-        => Fixture.TestSqlLoggerFactory.Clear();
+    {
+        Fixture.TestSqlLoggerFactory.Clear();
+        // Fixture.TestSqlLoggerFactory.SetTestOutputHelper(testOutputHelper);
+    }
+
+    [ConditionalTheory]
+    [MemberData(nameof(IsAsyncData))]
+    public virtual async Task Compare_row_values_equal_without_expansion(bool async)
```

Comment:
/cc to @maumar with :heart: 

This implements null semantics for PG row values (see and implementatin comments above if interested in the technicalitie - it's kinda cool). This could be useful if we ever get around to bringing row values to relational (https://github.com/dotnet/efcore/issues/26822).

/cc @smitpatel